src/config/petsConfig.ts

```diff
@@ -0,0 +1,15 @@
+interface Pet {
+  name: string;
+  image: string;
+  englishName: string; // for routing?
+}
+
+const pets: Pet[] = [
+  { name: 'Hund', image: 'src/assets/dog.png', englishName:'dog'},
+  { name: 'Kaninchen', image: 'src/assets/bunny.png', englishName: 'rabbit'},
+  { name: 'Katze', image: 'src/assets/dog.png', englishName: 'cat' }, // placeholder image
+  { name: 'Fisch', image: 'src/assets/bunny.png', englishName: 'fish' } // placeholder image
```

Comment:
Images doesn't get loaded in the project.
You can fix this by importing the images and then use them in the objects:

```
import yourImage from '@/assets/yourImage.png';

// ...
{ name: 'Hund', image: yourImage, englishName:'dog'}
// ...
```